app/Models/License.php

```diff
@@ -10,6 +10,7 @@
 use Illuminate\Database\Eloquent\SoftDeletes;
 use Illuminate\Support\Facades\Auth;
 use Illuminate\Support\Facades\Session;
+use phpDocumentor\Reflection\Types\Collection;
```

Comment:
This smells funny, are we sure this is right?

Reply:
I saw that right after I pushed. PHPStorm added that and I didn't notice it. 

:nose: bad smell lol